website/src/theme/DocPage/Layout/Main/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Main from '@theme-original/DocPage/Layout/Main'
+import Giscus from '@giscus/react'
+import { useColorMode } from '@docusaurus/theme-common'
+
+export default function MainWrapper({ children, ...props }) {
+  const { isDarkTheme } = useColorMode()
+
+  return (
+    <Main {...props}>
+      {children}
+      <div style={{ height: '32px' }} />
+      <div className='row'>
+        <div className='col'>
+          <Giscus
+            repo='software-mansion/protostar'
+            repoId='R_kgDOGw_HxA'
+            categoryId='DIC_kwDOGw_HxM4CBC8m'
+            category='Announcements'
```

Comment:
I think we should make a separate category for docs discussion, let's say `Documentation`. Polluting Announcements looks bad to me.

Reply:
> It is recommended to use a category with the Announcements type so that new discussions can only be created by maintainers and giscus.